ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/services/representative-suitability.service.ts

```diff
@@ -13,4 +13,9 @@ export class RepresentativeSuitabilityService {
         const responses = await this.client.getUserSuitabilityAnswers().toPromise();
         return responses.map(suitability => this.mapper.map(suitability));
     }
+
+    async updateSuitabilityAnswers(hearingId: string, answers: HearingSuitabilityAnswer[]): Promise<void> {
+        const response = await this.client.updateSuitabilityAnswers(hearingId, answers).toPromise();
```

Comment:
you can return the response straight away:
```typescript
return await this.client.updateSuitabilityAnswers(hearingId, answers).toPromise();
```

Reply:
Made changes